examples/8ball.c

```diff
@@ -0,0 +1,82 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <concord/discord.h>
+
+void on_ready(struct discord *client)
+{
+  const struct discord_user *bot = discord_get_self(client);
+
+  log_info("8ball-Bot succesfully connected to Discord as %s#%s!",
+           bot->username, bot->discriminator);
+}
+
+void eight_ball(struct discord *client, const struct discord_message *msg) {
+
+	if(msg->author->bot) return;
+
+	srand(time(0));//generate seed for randomizer
```

Comment:
`srand()` sets a global seed variable for generating random values, as such you only need to initialize that once if you are setting it up with `time(NULL)`! I suggest moving that to `main()`